src/config.ts

```diff
@@ -157,6 +158,8 @@ export default {
   ENABLE_ASYNC_STACK_TRACES,
   ENABLE_METRICS,
   ENABLE_GEOLOCATION_LOGGING: ENABLE_GEOLOCATION_LOGGING === "true",
+  ENABLE_GRAVITY_MARKETING_COLLECTIONS:
+    ENABLE_GRAVITY_MARKETING_COLLECTIONS === "true",
```

Comment:
does it makes sense to make it false by default instead?

Reply:
ah so this actually isn't a default, but an explicit boolean check: if the value of the env variable `ENABLE_GRAVITY_MARKETING_COLLECTIONS` is anything other than `true`, then the output from our config would be `false`.

If we removed the `ENABLE_GRAVITY_MARKETING_COLLECTIONS === "true",` we might run into something like `ENABLE_GRAVITY_MARKETING_COLLECTIONS=asdkfjhkjaldh`, which JavaScript would regard as truthy. Unlikely to happen, but seemed better to make it explicit :)

Reply:
makes sense, thank you for explanation! @mdole